docs/changelog.rst

```diff
@@ -11,6 +11,7 @@ Unreleased
 
 Added
 -----
+- write raster (DataArray) to tiles in xyz stucture
```

Comment:
Try to be more complete here to show all changes made in the PR

```suggestion
- write raster (DataArray) to tiles in xyz stucture
- add zoom_level to named arguments of path in the data catalog, to keep a single data source for the same data at different resolutions
- fix bug related to opening named raster files #200 
```

Reply:
Done